frontend/src/views/MovieAddScreen.vue

```diff
@@ -107,7 +107,7 @@ export default {
         };
 
         const { data: movie } = await axios.post(
-          `${import.meta.env.VITE_API_URL}/api/cinema/movies`,
+          `${import.meta.env.VITE_API_URL}/api/cinema/movies/`,
           {
             title: this.title,
```

Comment:
please review and test your project once more, a lot of endpoints are missed

Reply:
@Y-Havryliv I didn't get it, you mean I didn't show any screenshots of it, or what?

Reply:
I doubt your project works as expected, e.g. how do you upload an image using frontend?
Just re-check everything and make sure it works

Reply:
yes, it works,i have added transformers image using frontend :)